actors/builtin/market/market_actor.go

```diff
@@ -5,6 +5,7 @@ import (
 	"sort"
 
 	addr "github.com/filecoin-project/go-address"
+	"github.com/ipfs/go-cid"
```

Comment:
Nit: convention here is to always alias package names that don't match their final path segment.

Motivation: https://github.com/golang/go/issues/28428, https://go-review.googlesource.com/c/tools/+/152000

Reply:
This is done.